services/src/main/java/com/linkedin/kmf/services/configs/ProduceServiceConfig.java

```diff
@@ -26,6 +28,7 @@
 
   public static final String TOPIC_CONFIG = CommonServiceConfig.TOPIC_CONFIG;
   public static final String TOPIC_DOC = CommonServiceConfig.TOPIC_DOC;
+  public static final String TOPIC_CONFIG_DEFAULT = "kmf-producer";
```

Comment:
Probably CommonServiceConifg should have the TOPIC_CONFIG_DEFAULT.  It's not clear to me why the defaults are not constants already since they can exist in multiple Config classes.